README.md

```diff
@@ -45,6 +45,13 @@ It currently bridges PrivacyLx IRC <-> Matrix internal chat rooms. It can be
 extended to support more networks and bridges can be added by configuring
 `matterbridge/templates/matterbridge.toml.j2`.
 
+### Admin role
+
+This role is used to add/remove users, groups, permissions rights and access to
+hosts. You can deploy this role by running:
+
+`cd ansible && ansible-playbook -i inventory/production adm.yml`
```

Comment:
I don't think `adm.yml` is the best name for this. Why not `deploy-admin.yml` or `deploy-admin-users.yml` to conform with the others?

Reply:
Makes sense, I renamed the role `deploy-admin.yml` instead.